MAINTAINERS.md

```diff
@@ -1,11 +1,13 @@
-# OpenSearch JDBC Maintainers
+## Overview
 
-## Maintainers
+This document contains a list of maintainers in this repo. See [opensearch-project/.github/RESPONSIBILITIES.md](https://github.com/opensearch-project/.github/blob/main/RESPONSIBILITIES.md#maintainer-responsibilities) that explains what the role of maintainer means, what maintainers do in this and other repos, and how they should be doing it. If you're interested in contributing, and becoming a maintainer, see [CONTRIBUTING](CONTRIBUTING.md).
 
-| Maintainer | GitHub ID | Affiliation |
-| --------------- | --------- | ----------- |
-| Anirudha (Ani) Jadhav | [anirudha](https://github.com/anirudha) | Amazon |
-| Peng Huo | [penghuo](https://github.com/penghuo) | Amazon |
-| Chloe | [chloe-zh](https://github.com/chloe-zh) | Amazon |
-| Chen Dai | [dai-chen](https://github.com/dai-chen) | Amazon |
-| Harold Wang | [harold-wang](https://github.com/harold-wang) | Amazon |
+## Current Maintainers
+
+| Maintainer            | GitHub ID                                     | Affiliation |
+| --------------------- | --------------------------------------------- | ----------- |
+| Anirudha (Ani) Jadhav | [anirudha](https://github.com/anirudha)       | Amazon      |
+| Peng Huo              | [penghuo](https://github.com/penghuo)         | Amazon      |
+| Chloe                 | [chloe-zh](https://github.com/chloe-zh)       | Amazon      |
+| Chen Dai              | [dai-chen](https://github.com/dai-chen)       | Amazon      |
+| Harold Wang           | [harold-wang](https://github.com/harold-wang) | Amazon      |
```

Comment:
@YANG-DB  we should update this list to match the SQL repo: https://github.com/opensearch-project/sql/blob/main/MAINTAINERS.md

Reply:
ok